doc/api/toolkits/mplot3d/view_angles.rst

```diff
@@ -26,13 +26,26 @@ as well as roll, and all three angles can be set programmatically::
 
     import matplotlib.pyplot as plt
     ax = plt.figure().add_subplot(projection='3d')
-    ax.view_init(elev=30, azim=45, roll=15)
+    ax.view_init(azim=45, elev=30, roll=15)
+
+Rotation of the plot
+====================
```

Comment:
I think the discussion around quaternions might be confusing, as no quaternion interface is exposed to the user. Perhaps something like the following? I haven't thought through the rotation handedness so that might need to be changed.

```
The azim, elev, roll rotation order corresponds to rotation of the axes in a fixed camera view frame. A left-hand rotation about camera z is applied (azim), then a right hand rotation about camera y (elev), then a right hand rotation about camera x (roll).

This can also be thought of orbiting a camera around fixed axes by flipping the order of operations. First the camera is rotated clockwise about the scene's +x axis (roll), then the +y axis (elev), then the -z axis (azim).
```

Reply:
I added something along the lines of your suggestion; the connection with quaternions is still mentioned but less prominently. Take a look.